proteus/mprans/PresInc.pyx

```diff
@@ -443,12 +449,16 @@ class Coefficients(TC_base):
         """
         alphaBDF = self.fluidModel.timeIntegration.alpha_bdf
         for i in range(self.fluidModel.q[('velocity',0)].shape[-1]):
-            self.fluidModel.q[('velocity',0)][...,i] -= self.model.q[('grad(u)',0)][...,i]/(self.rho_f_min*alphaBDF)
+            #cek hack: for three-phase flow we'll have to do this differently
```

Comment:
for now I assume just air/water flow and am kicking the issue of how to correct the fluid and solid phase velocities in the case of an air/water/sediment mixture.